src/webhook/index.js

```diff
@@ -249,6 +149,55 @@ const singleUserHandler = async (
         el: otherFields.message.type,
       })
       .send();
+  } else if (type === 'postback') {
+    let input;
+
+    const postbackData = JSON.parse(otherFields.postback.data);
+
+    // Handle the case when user context in redis is expired
+    if (!context.data) {
+      lineClient.post('/message/reply', {
+        replyToken,
+        messages: [
+          {
+            type: 'text',
+            text: '🚧 ' + t`Sorry, the button is expired.`,
+          },
+        ],
+      });
+      clearTimeout(timerId);
+      return;
+    }
+
+    // When the postback is expired,
+    // i.e. If other new messages have been sent before pressing buttons,
+    // Don't do anything, just ignore silently.
```

Comment:
Outdated comment. Hope we can fix the comment along with this PR XD

```suggestion
    // tell the user about the expiry of buttons
```